build.yaml

```diff
@@ -188,7 +190,7 @@ common_cli:
     # permissions/owners.
     # This is to be tackled as part of:
     # https://clusterhq.atlassian.net/browse/FLOC-2689
-    sudo chmod 777 /var/run/docker.sock
+    test -e /var/run/docker.sock && sudo chmod 777 /var/run/docker.sock
 
```

Comment:
this was failling on OSX builds, don't attempt to remove a file which doesn't exist